app/models/student.rb

```diff
@@ -60,4 +62,11 @@ def check_justifications(klass, email)
   def justifications
     Justification.all.where(email: email)
   end
+
+  private
+
+  def attendance_info(klass)
+    name = klass.name.gsub(%r{\d{2}\/\d{2}\/\d{2,4}}, '').rstrip
```

Comment:
I have a suggestion: What do you think to remove this regex and create a script to update the database removing the `date` inside the `name`?